src/components/v5/shared/RichText/RichText.tsx

```diff
@@ -16,43 +21,51 @@ const RichText: FC<RichTextProps> = ({
   toggleOnDecriptionSelect,
   toggleOffDecriptionSelect,
 }) => {
-  const { editorContent, notFormattedContent, field } = useRichText(
-    name,
-    isDecriptionFieldExpanded,
-  );
+  const { editorContent, notFormattedContent, field, characterCount } =
+    useRichText(name, isDecriptionFieldExpanded);
   const { formatMessage } = useIntl();
 
-  const characterCount = editorContent?.storage.characterCount.characters();
-
   return (
     <>
       {editorContent && isDecriptionFieldExpanded ? (
         <>
           <MenuBar editor={editorContent} />
           <EditorContent editor={editorContent} {...field} />
 
-          {characterCount >= MIN_ANNOTATION_NUM &&
+          {(characterCount || isDecriptionFieldExpanded) && (
+            <div className="flex items-center justify-between mt-4">
+              {isDecriptionFieldExpanded && (
+                <TextButton
+                  mode="underlined"
+                  onClick={toggleOffDecriptionSelect}
+                >
+                  {formatMessage({ id: 'button.show.less' })}
+                </TextButton>
+              )}
+              {characterCount >= 1000 && isDecriptionFieldExpanded && (
+                <div className="text-xs text-gray-500 flex justify-end">
+                  {characterCount} / {MAX_ANNOTATION_NUM}
+                </div>
+              )}
+            </div>
+          )}
+        </>
+      ) : (
+        <>
+          <span
+            className={clsx({
+              'line-clamp-2 text-left': characterCount >= MIN_ANNOTATION_NUM,
+            })}
+          >
+            {notFormattedContent}
+          </span>
+          {characterCount > NUMBER_OF_CHARS_IN_TWO_LINES &&
             !isDecriptionFieldExpanded && (
               <TextButton mode="underlined" onClick={toggleOnDecriptionSelect}>
```

Comment:
`expand` button is gray-400 on design, on your screenshot it's different, please check this